mscore/data/shortcuts.xml

```diff
@@ -221,7 +221,7 @@
     <seq>Shift+O</seq>
     </SC>
   <SC>
-    <key>add-sforzato</key>
+    <key>add-accent</key>
```

Comment:
Will this change work correctly with existing users' shortcuts files? Shortcut file will contain Shift+V shortcut which references `add-sforzato` action which doesn't exist anymore, so the shortcut may happen to be lost at all.

Actually I don't know the answer, perhaps the new default will somehow take a precedence. But that probably needs testing.

Reply:
An existing shortcut for 'add-sforzato' will simply get ignored

Reply:
An existing **changed** shortcut for 'add-sforzato' will simply get ignored.
But the default shortcut Shift+V will continue to work.

Reply:
An existing **changed** shortcut for 'add-sforzato' will simply get ignored.
But the default shortcut Shift+V will continue to work (and map to add-accent), after a factory reset.
Without a factory reste, the old existing shortucst would get used and not have have this shortcut set up.

Reply:
On start I see "Debug: cannot find shortcut <add-sforzato> (...\MuseScore\mscore\shortcut.cpp:4167, static void Ms::Shortcut::load())". The shortcut `add-sforzato` will simple get ignored. The user would either need to set up a new shortcutr (e.g. "Shift+V" again) `for add-accent` or take a factory reset.